### ISSUE TITLE
List item having margin from left

### ISSUE THREAD

**Issue by tundak** [GeekyAnts/NativeBase#1274]
there is bug with `ListItem` having margin from left in Moto g 5.

**Comment by akhil-geekyants** [comment]
Is the margin present only on Moto g5. If it is present in every device you can see this issue #1219

**Comment by AntonPuko** [comment]
also, have unexpected left margin instead of padding in listItem. this doesn't look as expected behavior
![iml1-zxu-km](https://user-images.githubusercontent.com/8849922/31418250-4284dcb2-ae35-11e7-8527-cc9ccf684579.jpg)
phone: galaxy a5.
native-base: 2.3.2

**Comment by dragonfire1119** [comment]
This happens on the iOS Simulator as well. I'm just manually over riding the style with ``marginLeft: 0, paddingLeft: 17`` I'm on native-base: 2.3.2

**Comment by shivrajkumar** [comment]
@dragonfire1119 The margin is actually intended to have some space from the left for both the Text and its bottom border. Using padding doesn't provide that space/gap for the bottom border.
Use List with same background as the ListItem as the wrapper for those ListItems. It will resolve the issue.

**Comment by AntonPuko** [comment]
`Use List with same background as the ListItem as the wrapper for those ListItems. It will resolve the issue.`
Sorry, that I'm a bit rude, but its bad solution. Instead of figuring out how to structure you components right way, you just say: 'make a crunch' and close the issue. ListItem should definitely take full space of list.

**Comment by dragonfire1119** [comment]
@shivrajkumar Thanks for the reply! I understand you on the background & I already knew that. But I want a different background so it contrasts better instead of all white. Which to have a different back you have to use padding. Am I not getting how you designed it? If you designed it to only work with white background why is the background grey?

**Comment by Ewocker** [comment]
Same Issue here, this is definitely not expected behavior, margin on the left is really bugging me a lot.

**Comment by xyzzyxyzzy0077** [comment]
Same thing on iOS, should be a glitch

**Comment by ashkonhakimi** [comment]
I got it to work as follows.  

` width: '100%',
    marginLeft: 0,
    paddingLeft: 0,
    paddingRight: 0,
    marginRight: 0 `

**Comment by nonameolsson** [comment]
Is this something introduced in a newer version? I upgraded, and suddenly I started having this weird leftMargin. `paddingLeft` should be used.

**Comment by ashkonhakimi** [comment]
Strange. I have the newest version I did not run into that.

**Comment by nonameolsson** [comment]
This is how it looks out of the box now
<img width="505" alt="strange" src="https://user-images.githubusercontent.com/1493018/32250694-86c45bb0-be5b-11e7-8ec7-14e1677e913e.png">

Code:
```javascript
<List style={{ flex: 1 }}>
  <Separator>
    <Text>VISIT INFORMATION</Text>
  </Separator>

  <ListItem icon>
    <Left>
      <Icon name='person' />
    </Left>
    <Body>
      <Text>Name: {name}</Text>
    </Body>
  </ListItem>
  <ListItem>
    <Text style={{ paddingLeft: 42 }}>Age: {age}</Text>
  </ListItem>
  ...
</List>
```

This is how it looked before upgrading:
<img width="505" alt="good" src="https://user-images.githubusercontent.com/1493018/32250856-2d96d850-be5c-11e7-82dc-c4c48d5060a3.png">

To get the same layout I now have to change the code:
```javascript
<List style={{ flex: 1, backgroundColor: '#fff' }}>
  <Separator>
    <Text>VISIT INFORMATION</Text>
  </Separator>

  <ListItem icon>
    <Left>
      <Icon name='person' />
    </Left>
    <Body>
      <Text>Name: {name}</Text>
    </Body>
  </ListItem>
  <ListItem>
    <Text style={{ paddingLeft: 42 }}>Age: {age}</Text>
  </ListItem>
  ...
</List>
```

**Comment by ashkonhakimi** [comment]
lol Yea, I was getting that on the previous version too. Based on what I'm reading in forums, it's there on purpose. Super weird. It took me 3 hours to fix it (see previous post)

**Comment by SupriyaKalghatgi** [comment]
The margin is actually intended to have some space from the left
This is intended design
Hence no update on this part

**Comment by nonameolsson** [comment]
@SupriyaKalghatgi Why using margin? Would padding be better for styling?

**Comment by cmmartin** [comment]
This should not be closed. Margin needs to be replaced with padding. This component is unusable without this change.

**Comment by waheed25** [comment]
Here was my problem 
![screen shot 2018-03-16 at 1 11 30 pm](https://user-images.githubusercontent.com/25927790/37510072-bcdf4306-291b-11e8-8e91-41ecf53ecec1.png)
i resolved it my paddingLeft:0

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.1](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.1)
